apps/api/src/app/events/e2e/digest-events.e2e.ts

```diff
@@ -812,14 +826,110 @@ describe('Trigger event - Digest triggered events - /v1/events/trigger (POST)',
       _templateId: template._id,
       _subscriberId: subscriber._id,
     });
-
-    expect(messages[0].content).to.include(digests[0].payload.postId);
     expect(messages[1].content).to.include(digests[1].payload.postId);
+    expect(messages[2].content).to.include(digests[2].payload.postId);
+    const jobCount = await jobRepository.count({
+      _environmentId: session.environment._id,
+      _templateId: template._id,
+    });
+    const allJobs = await jobRepository.find({
+      _environmentId: session.environment._id,
+      _templateId: template._id,
+    });
```

Comment:
You don't need this call. 🙂

Reply:
Sorry, these statements were added for debugging tests and forgot to remove :)